site/role/manifests/all_in_one_pe.pp

```diff
@@ -3,6 +3,8 @@
   $webhook_username = hiera('webhook_username', fqdn_rand_string(10, '', 'username'))
   $webhook_password = hiera('webhook_password', fqdn_rand_string(20, '', 'password'))
 
+  include profile::base
```

Comment:
When users are adding things to manage with the base profile I don't want that to unexpectedly take out their PE master.  

They can add `profile::base` later if they want to manage their PE master the same way as all their other nodes.

Does that seem reasonable?